wallet/wallet-cli-lib/src/commands/helper_types.rs

```diff
@@ -101,12 +101,25 @@ pub fn format_pool_info(
     balance: Amount,
     block_height: BlockHeight,
     block_timestamp: BlockTimestamp,
+    decimals: u8,
```

Comment:
I prefer passing the chain config in such a situation. This is very obfuscated.

Reply:
fixed